README.md

```diff
@@ -97,31 +97,37 @@ Please change the directory after cloning the repository using the `cd <folder-n
 Next, install the dependencies by running the following command in the `react-play` repo:
 
 ```bash
-npm install
+yarn install
```

Comment:
Let's keep `npm install` too.

We can have `yarn install` first with the instruction as you added. Then we can add, alternatively you can use `npm install` but we prefer `yarn`.